build-deps

```diff
@@ -2,6 +2,8 @@
 export ZERO_AR_DATE=1
 export SOURCE_DATE_EPOCH=0
 
+export RNFC_BUILD_LIBRARIES=1
```

Comment:
We are not running gradle through there so it doesn't matter.
We could potentially get rid of NDK in android entirely, but keeping it this way makes it possible for consumer of this package to chose to use local build if they prefer.

Reply:
Removed in https://github.com/ExodusMovement/react-native-fast-crypto/commit/654fb483dcbd30bed8cebf25a5f882620d7354d9